llvm/test/tools/repo2obj/extern_weak_references.ll

```diff
@@ -0,0 +1,56 @@
+// Check the weak symbol binding: the output symbol is weak if all of the references to it are weak.
```

Comment:
As above, the .ll file extension is misleading.

Reply:
Changed to .json.